### PR TITLE
Announcing v1.0.0

### PR DESCRIPTION
**Pull request by jodosha**
🎉 

Tasks:

  - [x] Team page
  - [x] Security policy
  - [x] Blog post
  - [x] Bump version
  - [x] Update Getting Started Guide
  - [x] Update deploy instructions of Heroku

### CODE REVIEW

**Comment by cllns** [comment]
Updated some wording on the blog post and security page, but 👍 to me 🌸

**Comment by jodosha** [comment]
@cllns Thank you. Are the blog post changes involving the ML issue as well? They share the same intro.

File: source/guides/getting-started.md [hunk 1]
```diff
@@ -1078,7 +1073,7 @@ We can use the `routes` helper method that is available in our views and actions
 We can make a similar change in `apps/web/controllers/books/create.rb`:
 
 ```ruby
-redirect_to routes.books_path
+redirect_to routes.books_url
```
Comment (cllns) [inline source/guides/getting-started.md]: why did this change from `books_path` to `books_url`?

**Comment by cllns** [comment]
@jodosha just updated the mailing list file!

File: source/guides/getting-started.md (hunk 1 above)
Comment (jodosha) [inline source/guides/getting-started.md]: Redirects are preferrable with absolute URLs instead of relative.

**Pull request merged**